package.json

```diff
@@ -52,7 +52,14 @@
         "onCommand:aks.aksKubectlK8sLivezAPIEndpointCommands",
```

Comment:
I noticed that some gadgets as a wide output and it doesn't fit on the screen:

![image](https://user-images.githubusercontent.com/57147708/219362729-f9ea464a-c1af-4999-a676-ed470c335fd1.png)

I was wondering if we could add a scrollbar because currently it is difficult to see the whole output.